### PR TITLE
[flutter_tools] change service worker load to NOTICES

### LINKED ISSUE #57904: [web] Error in flutter_service_worker.js
## Steps to Reproduce

1. Create a Flutter Project with **web** enabled
2. Run `flutter build web --release`.

**Expected results:** 
No errors in the service worker file

**Actual results:** 
`add()` is used instead of `push()`
No method named `add()` exists for array in JavaScript
`Cache` variable used instead of `caches` and `addAll()` method doesn't exist in JavaScript (No variable named `Cache` exists)

Screenshot 2020-05-25 at 12 47 14 AM

<details>
  <summary>Logs</summary>

```
[  +11 ms] executing: [/Users/aseef/SDKs/flutter/] git -c
log.showSignature=false log -n 1 --pretty=format:%H
[  +45 ms] Exit code 0 from: git -c log.showSignature=false log -n 1
--pretty=format:%H
[        ] 9d58a87066d1da98eb06826918e4b90cc76ae0ef
[        ] executing: [/Users/aseef/SDKs/flutter/] git tag --contains HEAD
[ +187 ms] Exit code 0 from: git tag --contains HEAD
[   +3 ms] executing: [/Users/aseef/SDKs/flutter/] git describe --match
*.*.*-*.*.pre --first-parent --long --tags
[ +151 ms] Exit code 0 from: git describe --match *.*.*-*.*.pre --first-parent
--long --tags
[        ] 1.19.0-1.0.pre-143-g9d58a8706
[   +9 ms] executing: [/Users/aseef/SDKs/flutter/] git rev-parse --abbrev-ref
--symbolic @{u}
[  +11 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/master
[        ] executing: [/Users/aseef/SDKs/flutter/] git ls-remote --get-url
origin
[  +10 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +59 ms] executing: [/Users/aseef/SDKs/flutter/] git rev-parse --abbrev-ref
HEAD
[  +11 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] master
[   +5 ms] executing: sw_vers -productName
[  +13 ms] Exit code 0 from: sw_vers -productName
[        ] Mac OS X
[        ] executing: sw_vers -productVersion
[  +12 ms] Exit code 0 from: sw_vers -productVersion
[        ] 10.15.4
[        ] executing: sw_vers -buildVersion
[  +13 ms] Exit code 0 from: sw_vers -buildVersion
[        ] 19E287
[  +34 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping
update.
[   +8 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required,
skipping update.
[  +18 ms] executing: /Users/aseef/Library/Android/sdk/platform-tools/adb
devices -l
[   +7 ms] executing: /usr/bin/xcode-select --print-path
[   +7 ms] Exit code 0 from: /usr/bin/xcode-select --print-path
[        ] /Applications/Xcode.app/Contents/Developer
[   +1 ms] executing: /usr/bin/xcodebuild -version
[ +435 ms] Exit code 0 from: /usr/bin/xcodebuild -version
[        ] Xcode 11.4.1
           Build version 11E503a
[   +2 ms] executing: xcrun --find xcdevice
[   +9 ms] Exit code 0 from: xcrun --find xcdevice
[        ] /Applications/Xcode.app/Contents/Developer/usr/bin/xcdevice
[        ] executing: xcrun xcdevice list --timeout 2
[   +3 ms] /usr/bin/xcrun simctl list --json devices
[        ] executing: /usr/bin/xcrun simctl list --json devices
[  +41 ms] List of devices attached
[  +97 ms] {
             "devices" : {
               "com.apple.CoreSimulator.SimRuntime.iOS-13-2" : [
                 {
                   "availabilityError" : "runtime profile not found",
                   "dataPath" :
                   "\/Users\/aseef\/Library\/Developer\/CoreSimulator\/Devices\/
                   CE784987-C6EC-4E5B-8BD1-6C6816522563\/data",
                   "logPath" :
                   "\/Users\/aseef\/Library\/Logs\/CoreSimulator\/CE784987-C6EC-
                   4E5B-8BD1-6C6816522563",
                   "udid" : "CE784987-C6EC-4E5B-8BD1-6C6816522563",
                   "isAvailable" : false,
                   "deviceTypeIdentifier" :
                   "com.apple.CoreSimulator.SimDeviceType.iPhone-8-Plus",
                   "state" : "Shutdown",
                   "name" : "iPhone 8 Plus"
                 },
                 {
                   "availabilityError" : "runtime profile not found",
                   "dataPath" :
                   "\/Users\/aseef\/Library\/Developer\/CoreSimulator\/Devices\/
                   94D40E1A-0901-458C-A660-014FC45FF489\/data",
                   "logPath" :
                   "\/Users\/aseef\/Library\/Logs\/CoreSimulator\/94D40E1A-0901-
                   458C-A660-014FC45FF489",
                   "udid" : "94D40E1A-0901-458C-A660-014FC45FF489",
                   "isAvailable" : false,
                   "deviceTypeIdentifier" :
                   "com.apple.CoreSimulator.SimDeviceType.iPhone-11",
                   "state" : "Shutdown",
                   "name" : "iPhone 11"
                 },
                 {
                   "availabilityError" : "runtime profile not found",
                   "dataPath" :
                   "\/Users\/aseef\/Library\/Developer\/CoreSimulator\/Devices\/
                   A7FC7EB6-DF36-495C-8A78-B14293A3C9F8\/data",
                   "logPath" :
                   "\/Users\/aseef\/Library\/Logs\/CoreSimulator\/A7FC7EB6-DF36-
                   495C-8A78-B14293A3C9F8",
                   "udid" : "A7FC7EB6-DF36-495C-8A78-B14293A3C9F8",
                   "isAvailable" : false,
                   "deviceTypeIdentifier" :
                   "com.apple.CoreSimulator.SimDeviceType.iPhone-11-Pro",
                   "state" : "Creating",
                   "name" : "iPhone 11 Pro"
                 },
                 {
                   "availabilityError" : "runtime profile not found",
                   "dataPath" :
                   "\/Users\/aseef\/Library\/Developer\/CoreSimulator\/Devices\/
                   43396A0E-4D2A-481C-9B3B-BA12FDE6CC31\/data",
                   "logPath" :
                   "\/Users\/aseef\/Library\/Logs\/CoreSimulator\/43396A0E-4D2A-
                   481C-9B3B-BA12FDE6CC31",
                   "udid" : "43396A0E-4D2A-481C-9B3B-BA12FDE6CC31",
                   "isAvailable" : false,
                   "deviceTypeIdentifier" :
                   "com.apple.CoreSimulator.SimDeviceType.iPhone-11-Pro-Max",
                   "state" : "Creating",
                   "name" : "iPhone 11 Pro Max"
                 },
               ],
             }
           }
[+3084 ms] [
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "13.4.1 (17E8260)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPad8,12",
                        "identifier" : "9F03EA65-602B-4D9D-B0FF-D262D2EC131B",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.ipad-pro-12point9-4th-1",
                        "modelName" : "iPad Pro (12.9-inch) (4th generation)",
                        "name" : "iPad Pro (12.9-inch) (4th generation)"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "6.2 (17T256)",
                        "available" : true,
                        "platform" : "com.apple.platform.watchsimulator",
                        "modelCode" : "Watch5,4",
                        "identifier" : "E38F4806-A5E2-4281-B54F-7BDBD702B821",
                        "architecture" : "i386",
                        "modelUTI" : "com.apple.watch-series5-1",
                        "modelName" : "Apple Watch Series 5 - 44mm",
                        "name" : "Apple Watch Series 5 - 44mm"
                      },{
                        "simulator" : true,
                        "operatingSystemVersion" : "13.4.1 (17E8260)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone10,5",
                        "identifier" : "DA3C4D56-43D4-4035-A83E-EFE65CEDAFAE",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.iphone-8-plus-2",
                        "modelName" : "iPhone 8 Plus",
                        "name" : "iPhone 8 Plus"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "13.4.1 (17E8260)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone11,8",
                        "identifier" : "25DFA499-7A1C-40C4-9D9A-F6C675BBC7D1",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.iphone-xr-1",
                        "modelName" : "iPhone Xʀ",
                        "name" : "iPhone XR"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "13.4.1 (17E8260)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone12,5",
                        "identifier" : "20E825BA-3456-42E7-A0D2-DDD8872B570E",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.iphone-11-pro-max-1",
                        "modelName" : "iPhone 11 Pro Max",
                        "name" : "iPhone 11 Pro Max"
                      },
                      {
                        "simulator" : true,
                        "operatingSystemVersion" : "13.4.1 (17E8260)",
                        "available" : true,
                        "platform" : "com.apple.platform.iphonesimulator",
                        "modelCode" : "iPhone12,1",
                        "identifier" : "960D2BAC-4A4D-4B55-9F49-B3ECBF486EF3",
                        "architecture" : "x86_64",
                        "modelUTI" : "com.apple.iphone-11-1",
                        "modelName" : "iPhone 11",
                        "name" : "iPhone 11"
                      },
                    ]
[   +9 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping
update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping
update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required,
skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required,
skipping update.
[  +93 ms] Found plugin cloud_firestore at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.13.5/
[   +8 ms] Found plugin cloud_firestore_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore_web-0.1.1+2/
[   +6 ms] Found plugin firebase_auth at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.16.0/
[   +3 ms] Found plugin firebase_auth_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth_web-0.1.2/
[   +2 ms] Found plugin firebase_core at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.4.4+3/
[   +3 ms] Found plugin firebase_core_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core_web-0.1.1+2/
[   +2 ms] Found plugin fluttertoast at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/fluttertoast-3.1.3/
[   +2 ms] Found plugin google_sign_in at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in-4.5.1/
[   +2 ms] Found plugin google_sign_in_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in_web-0.9.1+1/
[  +59 ms] Found plugin cloud_firestore at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.13.5/
[   +2 ms] Found plugin cloud_firestore_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore_web-0.1.1+2/
[   +4 ms] Found plugin firebase_auth at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.16.0/
[   +2 ms] Found plugin firebase_auth_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth_web-0.1.2/
[   +1 ms] Found plugin firebase_core at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.4.4+3/
[   +2 ms] Found plugin firebase_core_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core_web-0.1.1+2/
[   +2 ms] Found plugin fluttertoast at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/fluttertoast-3.1.3/
[   +1 ms] Found plugin google_sign_in at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in-4.5.1/
[   +1 ms] Found plugin google_sign_in_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in_web-0.9.1+1/
[  +72 ms] Generating
/Users/aseef/Desktop/FlutterWebProjects/cigarette_track/android/app/src/main/jav
a/io/flutter/plugins/GeneratedPluginRegistrant.java
[ +161 ms] Launching lib/main.dart on Chrome in debug mode...
[  +60 ms] Updating assets
[ +153 ms] Manifest contained wildcard assets. Inserting missing file into build
graph to force rerun. for more information see #56466.
[   +5 ms] Syncing files to device Chrome...
[   +7 ms] Found plugin cloud_firestore at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.13.5/
[   +1 ms] Found plugin cloud_firestore_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore_web-0.1.1+2/
[   +2 ms] Found plugin firebase_auth at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.16.0/
[   +1 ms] Found plugin firebase_auth_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth_web-0.1.2/
[        ] Found plugin firebase_core at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.4.4+3/
[   +1 ms] Found plugin firebase_core_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core_web-0.1.1+2/
[   +1 ms] Found plugin fluttertoast at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/fluttertoast-3.1.3/
[        ] Found plugin google_sign_in at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in-4.5.1/
[   +1 ms] Found plugin google_sign_in_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in_web-0.9.1+1/
[  +17 ms] Found plugin cloud_firestore at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore-0.13.5/
[   +1 ms] Found plugin cloud_firestore_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/cloud_firestore_web-0.1.1+2/
[   +2 ms] Found plugin firebase_auth at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.16.0/
[   +1 ms] Found plugin firebase_auth_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_auth_web-0.1.2/
[   +1 ms] Found plugin firebase_core at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core-0.4.4+3/
[   +1 ms] Found plugin firebase_core_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/firebase_core_web-0.1.1+2/
[   +1 ms] Found plugin fluttertoast at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/fluttertoast-3.1.3/
[        ] Found plugin google_sign_in at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in-4.5.1/
[   +1 ms] Found plugin google_sign_in_web at
/Users/aseef/.pub-cache/hosted/pub.dartlang.org/google_sign_in_web-0.9.1+1/
[  +15 ms] Generating
/Users/aseef/Desktop/FlutterWebProjects/cigarette_track/android/app/src/main/jav
a/io/flutter/plugins/GeneratedPluginRegistrant.java
[  +45 ms] <- reset
[   +6 ms] /Users/aseef/SDKs/flutter/bin/cache/dart-sdk/bin/dart
/Users/aseef/SDKs/flutter/bin/cache/artifacts/engine/darwin-x64/frontend_server.
dart.snapshot --sdk-root /Users/aseef/SDKs/flutter/bin/cache/flutter_web_sdk/
--incremental --target=dartdevc --debugger-module-names
-Ddart.developer.causal_async_stacks=true --output-dill
/var/folders/1k/vg7lwkkd5_35y7s476gn723h0000gn/T/flutter_tools.XZ0Emw/flutter_to
ol.bramao/app.dill --libraries-spec
file:///Users/aseef/SDKs/flutter/bin/cache/flutter_web_sdk/libraries.json
--packages .packages -Ddart.vm.profile=false -Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field
-initializers,keep-unreachable-code,avoid-closure-call-instructions
--enable-asserts --track-widget-creation --filesystem-root
/var/folders/1k/vg7lwkkd5_35y7s476gn723h0000gn/T/flutter_tools.XZ0Emw/flutter_to
ols.ZbC00I --filesystem-scheme org-dartlang-app --initialize-from-dill
build/cache.dill.track.dill --platform
file:///Users/aseef/SDKs/flutter/bin/cache/flutter_web_sdk/kernel/flutter_ddc_sd
k.dill
[   +7 ms] <- compile org-dartlang-app:/web_entrypoint.dart
[+17646 ms] Syncing files to device Chrome... (completed in 17,758ms, longer
than expected)
[   +1 ms] Synced 26.8MB.
[        ] <- accept
[ +488 ms] [CHROME]:
[   +1 ms] [CHROME]:DevTools listening on
ws://127.0.0.1:63840/devtools/browser/c7790d0b-100a-4453-a71c-e93c3112c7a9
[+3489 ms] Debug service listening on ws://127.0.0.1:63949/XkNhZFZv2X0=

[ +566 ms] Debug service listening on ws://127.0.0.1:63949/XkNhZFZv2X0=
[   +2 ms] Warning: Flutter's support for web development is not stable yet and
hasn't
[        ] been thoroughly tested in production environments.
[        ] For more information see https://flutter.dev/web
[        ] 🔥  To hot restart changes while running, press "r" or "R".
[   +1 ms] For a more detailed help message, press "h". To quit, press "q".
[  +82 ms] WARNING: found an existing <meta name="viewport"> tag. Flutter Web
uses its own
           viewport configuration for better compatibility with Flutter. This
tag will be
           replaced.
           
```

```
Analyzing web_test...                                            
No issues found! (ran in 4.7s)
```

```
[✓] Flutter (Channel master, 1.19.0-2.0.pre.143, on Mac OS X 10.15.4 19E287,
    locale en-GB)
    • Flutter version 1.19.0-2.0.pre.143 at /Users/aseef/SDKs/flutter
    • Framework revision 9d58a87066 (2 days ago), 2020-05-22 22:37:01 -0700
    • Engine revision 9ce1e5c5c7
    • Dart version 2.9.0 (build 2.9.0-10.0.dev 7706afbcf5)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/aseef/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4.1, Build version 11E503a
    • CocoaPods version 1.8.4

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.45.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.10.2

[✓] Connected device (2 available)
    • Web Server • web-server • web-javascript • Flutter Tools
    • Chrome     • chrome     • web-javascript • Google Chrome 81.0.4044.138

• No issues found!
```

</details>

### LINKED ISSUE #58888: Missing LICENSE file prevents service worker installation
Hello, 

In previous versions there was the possibility to install a web application in Google Chrome, as shown in the image:

install

But in more recent versions compiling the same application, this option is no longer visible:

recent

Is there any way to activate this feature when compiling the application?

```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, 1.19.0-4.0.pre.92, on Microsoft Windows [VersiÃ³n 10.0.18363.836], locale es-CO)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.4.0)
[√] Android Studio (version 3.6)
[√] VS Code (version 1.45.1)
[√] Connected device (3 available)

• No issues found!
```

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Did not update flutter_service_worker.js when updating LICENSE to NOTICES

Fixes https://github.com/flutter/flutter/issues/58888
Fixes https://github.com/flutter/flutter/issues/57904

### CODE REVIEW

File: packages/flutter_tools/lib/src/build_system/targets/web.dart [hunk 1]
```diff
@@ -543,7 +543,7 @@ async function downloadOffline() {
   }
   for (var resourceKey in Object.keys(RESOURCES)) {
     if (!currentContent[resourceKey]) {
-      resources.add(resourceKey);
+      resources.push(resourceKey);
     }
   }
   return Cache.addAll(resources);
```
Comment (azbuky) [inline packages/flutter_tools/lib/src/build_system/targets/web.dart:549]: @jonahwilliams 
This line also causes an error when installing the service worker.

```suggestion
  return contentCache.addAll(resources);
```

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/build_system/targets/web.dart]: thanks!

File: packages/flutter_tools/lib/src/build_system/targets/web.dart [hunk 2]
```diff
@@ -543,10 +543,10 @@ async function downloadOffline() {
   }
   for (var resourceKey in Object.keys(RESOURCES)) {
     if (!currentContent[resourceKey]) {
-      resources.add(resourceKey);
+      resources.push(resourceKey);
```
Comment (christopherfujino) [inline packages/flutter_tools/lib/src/build_system/targets/web.dart:546]: what data type is this?

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/build_system/targets/web.dart:546]: its a JavaScript Array

Comment (christopherfujino) [reply packages/flutter_tools/lib/src/build_system/targets/web.dart:546]: ahhh, ok, I see it now!

**Review by christopherfujino — APPROVED** [review]
lgtm

**Pull request merged**